core/src/main/kotlin/com/trevjonez/composer/ComposerTask.kt

```diff
@@ -77,10 +77,14 @@ open class ComposerTask : JavaExec(), ComposerConfigurator, ComposerTaskDsl {
       }
     }
 
+    val apks: List<File> = project.configurations.findByName("androidTestUtil")?.resolvedConfiguration?.files?.toList()
```

Comment:
lets make it a lazy property similar to the apk and testApk.
then we can also move the default behavior of looking for androidTestUtil into the plugin since the core module doesn't directly depend on the AGP and androidTestUtil is an AGP thing.

I have been trying to keep core in the form of "bucket of vitamins anyone can use and customize" and plugin is "make it as easy as possible on the user"